finalPrep/modZeroHero.js

```diff
@@ -74,11 +148,50 @@ var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.
 
 // - Create 2 instances of your SuperHero class
 
+class SuperHero {
+  constructor(name, superpower, age, archNemesis, powerLevel, energyLevel) {
```

Comment:
Because `archNemesis`, `powerLevel`, and `energyLevel` are to be assigned to static values, they don't need to be passed into the constructor function.